### PR TITLE
[TEACH-456] Section Setup UI Polish

### PR DESCRIPTION
**Pull request by levadadenys**
### [TEACH-456] Section Setup UI Polish

Before:
image
image

After:
image

image

## Links
- jira ticket: [TEACH-456](https://codedotorg.atlassian.net/browse/TEACH-456?atlOrigin=eyJpIjoiZWYxYzc5ODc3MGNmNDQ1Y2E3N2Y4MzBkOWMxY2UxNjEiLCJwIjoiaiJ9)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/sectionsRefresh/CurriculumQuickAssignTopRow.jsx [hunk 1]
```diff
@@ -12,84 +13,96 @@ export const MARKETING_AUDIENCE = {
...
-            e.preventDefault();
-            determineMarketingAudience(MARKETING_AUDIENCE.HOC);
-          }}
         />
+
         {showPlOfferings && (
-          <Button
-            id={'uitest-pl-button'}
-            className={moduleStyles.buttonStyle}
+          <MarketingAudienceButton
```
Comment (hannahbergam) [inline apps/src/templates/sectionsRefresh/CurriculumQuickAssignTopRow.jsx:101]: I love how clean the render is after the refactor! Thank you for that update!

**Review by hannahbergam — APPROVED** [review]
I love the highlighting for the selected top row buttons. This is just personal preference (and likely because I've been staring at the more spread out version for weeks) but now I think the space between lines on the table looks a little small. But that's just a nit! Thanks for all your work on this!

File: apps/src/templates/sectionsRefresh/CurriculumQuickAssignTopRow.jsx (hunk 1 above)
Comment (levadadenys) [reply apps/src/templates/sectionsRefresh/CurriculumQuickAssignTopRow.jsx:101]: Thanks for the feedback!
Re size of the lines between table columns: Looks like they are similar as they show on the designs and readliens, so I guess it's ok to leave it as it right now.
@molly-moen correct me if I'm wrong

Comment (molly-moen) [reply apps/src/templates/sectionsRefresh/CurriculumQuickAssignTopRow.jsx:101]: I'm guessing you meant to tag Molly Peredo? I can't find her on github to tag in.

Comment (levadadenys) [reply apps/src/templates/sectionsRefresh/CurriculumQuickAssignTopRow.jsx:101]: Omg, I'm sorry Molly! Yeah, I've meant Molly Peredo. I'll send her a link in slack!

**Pull request merged**